parameter_files/fates_params_default.cdl

```diff
@@ -1020,7 +1020,8 @@ data:
 
  fates_damage_recovery_scalar = 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ;
 
- fates_dev_arbitrary_pft = _, _, _, _, _, _, _, _, _, _, _, _ ;
+ fates_dev_arbitrary_pft = 1.7560, 1.4995, 1.4995, 1.7560, 1.7560, 1.7560, 
```

Comment:
@ckoven what would we call this parameter?  I see it matches r_0: "! base respiration rate, PFT-dependent (umol CO2/m**2/s)"

Also, is there a similar parameter in the Ryan model that could also leverage the parameter?

I could add this parameter to the nutrient coupling PR (which has other parameter changes, and assuming it gets through queue more quickly).

Reply:
Good question, and good idea.  Maybe `fates_maintresp_atkinetal2017model_baserate`?  There isn't a direct analogue in the Ryan 1991 model.

Reply:
how about: fates_maintresp_atkin2017_baserate

Reply:
@rgknox that works for me, thanks!